Build.mak

```diff
@@ -37,4 +37,5 @@ $O/test-httpserver: override LDFLAGS += -lglib-2.0
 
 # Link unittests to all used libraries
 $O/%unittests: override LDFLAGS += -lglib-2.0 -lpcre -lxml2 -lxslt -lebtree \
-		-lreadline -lhistory -llzo2 -lbz2 -lz -ldl -lgcrypt -lgpg-error -lrt
+		-lreadline -lhistory -llzo2 -lbz2 -lz -ldl -lgcrypt -lgpg-error -lrt \
+		-lssl -lcrypto
```

Comment:
Oh god, is this a breaking change or not :-/.

Reply:
If worse comes to worse, we could mark it as `+breaking`.

The reason is that ocean users do compile ocean unittests (but they don't run them) in their unittest target IIRC and I wonder if this will make them update their dependencies.

Thinking again, I don't think it will be an issue, as I never used `-lhistory` in my Build.maks

Reply:
Yeah. Well actually there are no unittests, so it isn't technically required. But I think it is bad form to include the bindings without documenting what linker flags are required.

Reply:
@nemanja-boric-sociomantic : IIRC Makd only includes the imported modules. As no part of Ocean depends on this, it should be fine. But feel free to test :)